temp/legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/components/CurrencyPicker.kt

```diff
@@ -66,6 +67,7 @@ import java.util.Locale
 fun CurrencyPicker(
     modifier: Modifier = Modifier,
     initialSelectedCurrency: IvyCurrency?,
+    manualExchangeRates: List<ExchangeRate> = listOf(),
```

Comment:
Ideally, the entire currency list (standard + manual) should come from the VM and not he computed in the UI. Also, your fix will work for Settings but not for the account creation flow